src/Admin/SupportMenu.php

```diff
@@ -0,0 +1,250 @@
+<?php
+/**
+ * Class SupportMenu
+ *
+ * @package Ampproject\Ampwp
+ */
+
+namespace AmpProject\AmpWP\Admin;
+
+use AmpProject\AmpWP\Infrastructure\Conditional;
+use AmpProject\AmpWP\Infrastructure\Registerable;
+use AmpProject\AmpWP\Infrastructure\Service;
+use AmpProject\AmpWP\Services;
+
+/**
+ * SupportMenu class.
+ *
+ * @internal
+ */
+class SupportMenu implements Conditional, Service, Registerable {
+
+	/**
+	 * Handle for JS file.
+	 *
+	 * @var string
+	 */
+	const ASSET_HANDLE = 'amp-support';
+
+	/**
+	 * AJAX action name to use.
+	 *
+	 * @var string
+	 */
+	const AJAX_ACTION = 'amp_send_support_request';
+
+	/**
+	 * The parent menu slug.
+	 *
+	 * @var string
+	 */
+	private $parent_menu_slug;
+
+	/**
+	 * GoogleFonts instance.
+	 *
+	 * @var GoogleFonts
+	 */
+	private $google_fonts;
+
+	/**
+	 * Class constructor.
+	 *
+	 * @param OptionsMenu $options_menu An instance of the class handling the parent menu.
+	 * @param GoogleFonts $google_fonts An instance of the GoogleFonts service.
+	 */
+	public function __construct( OptionsMenu $options_menu, GoogleFonts $google_fonts ) {
+
+		$this->parent_menu_slug = $options_menu->get_menu_slug();
+
+		$this->google_fonts = $google_fonts;
+	}
+
+	/**
+	 * Check whether the conditional object is currently needed.
+	 *
+	 * @return bool Whether the conditional object is needed.
+	 */
+	public static function is_needed() {
+
+		if ( ! is_admin() ) {
+			return false;
+		}
+
+		/**
+		 * Filter whether to enable the AMP settings.
+		 *
+		 * @param bool $enable Whether to enable the AMP settings. Default true.
+		 *
+		 * @since 0.5
+		 */
+		return (bool) apply_filters( 'amp_support_menu_is_enabled', true );
```

Comment:
I don't think this filter is needed.

```suggestion
		return is_admin();
```